Monika After Story/game/zz_backgrounds.rpy

```diff
@@ -0,0 +1,518 @@
+        unlocked=True,
+
+        #Programming points for the spaceroom
+        entry_pp=store.mas_background._def_background_entry,
+        exit_pp=store.mas_background._def_background_exit
+    )
+
+
+#START: Image definitions
+#Spaceroom
+image monika_day_room = "mod_assets/location/spaceroom/spaceroom.png"
+image monika_room = "mod_assets/location/spaceroom/spaceroom-n.png"
+#Thanks Orca
+image monika_rain_room = "mod_assets/location/spaceroom/spaceroom_rain.png"
+#Thanks Velius/Orca
+image monika_snow_room_day = "mod_assets/location/spaceroom/spaceroom_snow.png"
+image monika_snow_room_night = "mod_assets/location/spaceroom/spaceroom_snow-n.png"
+
+#TODO: locking/unlocking of this based on other backgrounds
+#START: Location Selector
+#init 5 python:
+#    # available only if moni affection is affectionate+
+#    addEvent(
+#        Event(
+#            persistent.event_database,
+#            eventlabel="monika_change_background",
+#            category=["location"],
+#            prompt="Can we go somewhere else?",
+#            pool=True,
+#            unlocked=False,
+#            rules={"no unlock": None},
+#            aff_range=(mas_aff.AFFECTIONATE, None)
+#        )
+#    )
+
+label monika_change_background:
+
+    m 1hua "Sure!"
+
+label monika_change_background_loop:
+
+    show monika 1eua at t21
+
+    $ renpy.say(m, "Where would you like to go?", interact=False)
+
+    python:
+        # build menu list
+        import store.mas_background as mas_background
+        import store.mas_moods as mas_moods
+
+        # we assume that we will always have more than 1
+        # default should always be at the top
+        backgrounds = [(mas_background_def.prompt, mas_background_def, False, False)]
+
+        # build other backgrounds list
+        other_backgrounds = [
+            (mbg_obj.prompt, mbg_obj, False, False)
+            for mbg_id, mbg_obj in mas_background.BACKGROUND_MAP.iteritems()
+            if mbg_id != "spaceroom" and mbg_obj.unlocked
+        ]
+
+        # sort other backgrounds list
+        other_backgrounds.sort()
+
+        # build full list
+        backgrounds.extend(other_backgrounds)
+
+        # now add final quit item
+        final_item = (mas_background.BACKGROUND_RETURN, False, False, False, 20)
+
+    # call scrollable pane
+    call screen mas_gen_scrollable_menu(backgrounds, mas_moods.MOOD_AREA, mas_moods.MOOD_XALIGN, final_item)
+
+    $ sel_background = _return
+
+    show monika at t11
+
+    # return value False? then return
+    if sel_background is False:
+        m 1eka "Oh, alright."
+        m "If you want to go somewhere, just ask, okay?"
+        return
+
+    if sel_background == mas_current_background:
+        m 1hua "We're here right now, silly."
+        m "Try again~"
+        jump monika_change_background_loop
+
+    $ skip_outro = False
+    $ skip_leadin = False
+
+    # otherwise, we can change the background now
+    if not skip_leadin:
```

Comment:
`skip_leadin` is set to False right before this, so this always happens. Is there a reason `skip_leadin` is set to False right before here?